app/src/main/res/values/strings.xml

```diff
@@ -50,4 +50,8 @@
     <string name="error_code_length">An invite code should be 6 digits long!</string>
     <string name="tell_us">Tell us about yourself!</string>
     <string name="footer_register">Have a DoSomething.org account? Sign in</string>
+    <string name="crop_photo">crop photo</string>
+    <string name="report_back_caption">Caption your photo</string>
+    <string name="report_back_number">Number of protesters attended</string>
```

Comment:
FIXME: This string should be dynamically built from the campaign's report back verb and noun.